app/controllers/ReportingController.scala

```diff
@@ -297,14 +299,17 @@ trait ReportingController extends BaseController {
     val reports =
       for {
         applications <- reportingRepository.onlineTestPassMarkReport
+        siftResults <- applicationSiftRepository.findAllResults
         fsacResults <- assessmentScoresRepository.findAll
         questionnaires <- questionnaireRepository.findForOnlineTestPassMarkReport(applications.map(_.applicationId))
       } yield {
         for {
-          a <- applications
-          fr = fsacResults.find(_.applicationId == UniqueIdentifier(a.applicationId))
-          q <- questionnaires.get(a.applicationId)
-        } yield OnlineTestPassMarkReportItem(ApplicationForOnlineTestPassMarkReportItem.create(a, fr), q)
+          application <- applications
+          appId = UniqueIdentifier(application.applicationId)
+          fr = fsacResults.find(_.applicationId == appId)
+          sr = siftResults.find(_.applicationId == application.applicationId)
```

Comment:
Lets not propagate this bad variable naming, can these be readableVariableNames please

Reply:
Already fixed in a PR coming off the back of this